src/index.js

```diff
@@ -0,0 +1,16 @@
+import './style.css';
+
+const myTodo = [
+  { description: 'Going to the GYM', completed: true, index: 1 },
+  { description: 'Cleaning my Room', completed: false, index: 2 },
+  { description: 'Washing my Car', completed: true, index: 3 },
+  { description: 'Coding for 4 hours', completed: false, index: 4 },
+];
+const list = document.getElementById('list');
+myTodo.forEach((value) => {
+  const li = document.createElement('li');
+  li.innerHTML = `
+        ${value.description}<i class="fa-solid fa-ellipsis-vertical"></i>
+    `;
+  list.appendChild(li);
+});
```

Comment:
- [ ] Please make sure to sort the list of todos according to their indexes. As per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.